lib/Controller/SAMLController.php

```diff
@@ -197,7 +212,7 @@ public function login($idp) {
 				// And base64 encode it
 				$data = base64_encode($data);
 
-				$response->addCookie('saml_data', $data, null, 'None');
+				$response->addCookie('saml_data', $data, 1800, 'None');
```

Comment:
```suggestion
				$expiry = new \DateTime();
				$expiry->add(new \DateInterval('PT30M'));
				$response->addCookie('saml_data', $data, $expiry, 'None');
```

Reply:
ha ah shit you are right

Reply:
Let me set it to null again actually to prevent issues.